tests/playwright-test/golden.spec.ts

```diff
@@ -80,6 +144,22 @@ test('should write missing expectations locally', async ({runInlineTest}, testIn
   expect(data.toString()).toBe('Hello world');
 });
 
+test('should write missing expectations locally with negated matcher', async ({runInlineTest}, testInfo) => {
+  const result = await runInlineTest({
+    'a.spec.js': `
+      const { test } = pwt;
+      test('is a test', ({}) => {
+        expect('Hello world').not.toMatchSnapshot('snapshot.txt');
```

Comment:
I don't think this should write the expectation file, because next time you run the test it will match and the test will fail.

Reply:
That's rational. Then the only way to use `not.toMatchSnapshot` will be by reusing a snapshot created by something else.